lib/ship.rb

```diff
@@ -0,0 +1,17 @@
+class Ship
+  attr_accessor :name, :length, :health
+
+  def initialize(name,length)
```

Comment:
Is it necessary that these are required params? Kinda makes sense, but maybe there are default values which make more sense?